README.md

```diff
@@ -1,24 +1,144 @@
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 Blog - Ruby on Rails - ROR Microverse Project Module 5<a name="about-project"></a>
 
-* Deployment instructions
+> Blog - Ruby on Rails developed with Ruby on Rails.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/Ruby_(programming_language)">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Migration DB feature: tables and Relations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template, follow these steps.
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/chelobotix/Blog-Ruby-on-Rails.git
+```
+
+To install the necessary dependencies, you should run:
+
+### `gem install bundler`
+
+### `bundler install`
+
+## 🤖 Available Scripts <a name="available-scripts"></a>
+
+In the project directory, you can run:
+
+### `rubocop -A`
+
+To fix all Linter error automatically
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Author1**
+
+: Marcelo Alarcon Barrenechea
+
+🇧🇴 Bolivia 💓
+
+- GitHub: [@chelobotix](https://github.com/chelobotix)
+- Linkedin: [Marcelo Alarcon](https://www.linkedin.com/in/marceloalarconb/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **More Features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/chelobotix/school-library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please let me know
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+My mother. Let's play until the end together ❤️ with 🧠
+<br/>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Is there any restriction to adapting the app to other porpuse??**
+
+  - No problem.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](<a href="./LICENSE">License</a>) licensed.
+
+<a href="./LICENSE">License</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+Original design idea by <a href="https://www.behance.net/sakwadesignstudio">Nelson Sakwa on Behance</a>.
```

Comment:
- [ ] [OPTIONAL] I would strongly recommend keeping only information related to your project and removing the unnecessary comment like this one and also your header section comment in order to keep you readme more professional 👍

Reply:
Thanks, @codecaiine. I have implemented the update.

Reply:
Thanks, @codecaiine. I have implemented the update.